catalog-rest-service/src/main/resources/json/schema/entity/teams/role.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
@harshach

Minimal schema for Role entity to unblock #1940 
Can be extended later to add more fields as needed.